### ISSUE TITLE
Argument error when DTASOF = 00000000

### ISSUE THREAD

**Issue by tuping** [annacruz/ofx#11]
Hello @annacruz 

I have tried to parse an .ofx generated directly from a famous brazilian bank (the big red one) that has the folowing content:

````xml
...
<LEDGERBAL>
<BALAMT>0,00
<DTASOF>00000000
</LEDGERBAL>
...
````

I can't upload the whole file because it is not mine, but the problem is that DTASOF can't be parsed:

````
ArgumentError: argument out of range
	from /.rvm/rubies/ruby-1.9.3-p545/lib/ruby/1.9.1/time.rb:202:in `local'
	from /.rvm/rubies/ruby-1.9.3-p545/lib/ruby/1.9.1/time.rb:202:in `make_time'
	from /.rvm/rubies/ruby-1.9.3-p545/lib/ruby/1.9.1/time.rb:271:in `parse'
	from /.rvm/gems/ruby-1.9.3-p545@ror/gems/ofx-0.3.2/lib/ofx/parser/ofx102.rb:109:in `build_date'
	from /.rvm/gems/ruby-1.9.3-p545@ror/gems/ofx-0.3.2/lib/ofx/parser/ofx102.rb:118:in `build_balance'
	from /.rvm/gems/ruby-1.9.3-p545@ror/gems/ofx-0.3.2/lib/ofx/parser/ofx102.rb:58:in `build_account'
	from /.rvm/gems/ruby-1.9.3-p545@ror/gems/ofx-0.3.2/lib/ofx/parser/ofx102.rb:26:in `account'
	from (irb):2:in `block in irb_binding'
	from /.rvm/gems/ruby-1.9.3-p545@ror/gems/ofx-0.3.2/lib/ofx.rb:25:in `instance_eval'
	from /.rvm/gems/ruby-1.9.3-p545@ror/gems/ofx-0.3.2/lib/ofx.rb:25:in `OFX'
	from (irb):2
	from /.rvm/rubies/ruby-1.9.3-p545/bin/irb:12:in `<main>'
````

**Comment by felipediesel** [comment]
@annacruz This problem still exists. Probably the wrong one here is Bradesco, but I think it should be fixed.

I imagined 3 possible solutions to solve this issue:

1) build_balance returns nil 
2) Set OFX::Balance :posted_at to nil
3) In build_date wrap Time.parse(date) with begin rescue and return nil if it couldn't be parsed.

I can fix this and send a pull request. But it would be nice to know what you think?

**Comment by annacruz** [comment]
@felipediesel I really know that's a little bit old, but can you send a PR with your fix? I like the solution and understood the problem =D

**Comment by felipediesel** [comment]
@annacruz We also implemented OFX 1.0.3 that is used by HSBC. Can I do a PR with it also? 

You can check the commit here: https://github.com/felipediesel/ofx/commit/f9dc2a959ea0ddc72db062db7c53b6c5c398d6be

It is the same parser as 1.0.2, just changes the VERSION.

**Comment by annacruz** [comment]
@felipediesel Yes! You can also do a PR with this update =)
